presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -129,16 +126,16 @@ void PeriodicTaskManager::start() {
         "clean_old_tasks");
   }
 
-  if (auto* mappedMemory = velox::memory::MappedMemory::getInstance()) {
+  if (auto* allocator = velox::memory::MemoryAllocator::getInstance()) {
```

Comment:
getInstance() guaranteed to be non-null. I think we might want to cast and check if the impl is mmap. Otherwise it can crash if the system is not running using malloc impl version

Reply:
There is a refactoring about this: https://www.internalfb.com/diff/D41568191